### ISSUE TITLE
The Glob / Beastie Feastie - "Cheat" button non functional, bad control defaults, double-listing

### ISSUE THREAD

**Issue by M-Walrus** [MiSTer-devel/Arcade-Pacman_MiSTer#43]
- Cheat button seems to not work in either version.

- Control defaults are not good.  It uses the P1 Start and P2 start buttons as the main action buttons.  Recommend the mra be updated to place these two buttons in the A/B location, or at least mirror them to A/B as well as have them in the standard P1/P2 start locations.

- Do we need both versions (The Glob *and* Beastie Feastie) of this listed in the main arcade .mra list, as well as a (pac-man conversion) options? 
They are the same game and all besides one should be removed and put in an alternate folder, for better organization.

**Comment by sorgelig** [comment]
Button functions are how they designed in arcade. Nothing wrong with core.

**Comment by tjanas** [comment]
> the .mra needs to be updated. The original machine had 4 buttons (2 action buttons and 1/2p start). Since this is the pac-man conversion, it uses the 1p/2p start buttons for action buttons also. Currently when binding buttons you have: Fire - Useless/non-functional and should be removed. 1p Start - Should be renamed "Call / 1P" 2p Start - Should be renamed "Energy / 2P" Cheat - Useless/non-functional and should be removed.

I agree. Labeling the inputs like this (and removing the unused Fire & Cheat inputs) makes sense for Beastie Feastie.
